dts/riscv/ite/it8xxx2.dtsi

```diff
@@ -306,6 +331,67 @@
 				volt-sel-mask = <0x10     0x10     0x10     0x10
 						 0x10     0x10     0x10     0       >;
 				#pinmux-cells = <2>;
+				gpio-group;
+			};
+
+			pinctrlksi: pinctrl@f01d06 {
+				compatible = "ite,it8xxx2-pinctrl-func";
+				reg = <0x00f01d06 1   /* KSIGCTRL */
+				       0x00f01d05 1>; /* KSICTRL */
+				func3-gcr =     <NO_FUNC NO_FUNC NO_FUNC NO_FUNC
+						 NO_FUNC NO_FUNC NO_FUNC NO_FUNC>;
+				func3-en-mask = <0       0       0       0
+						 0       0       0       0      >;
+				func4-gcr =     <NO_FUNC NO_FUNC NO_FUNC NO_FUNC
+						 NO_FUNC NO_FUNC NO_FUNC NO_FUNC>;
+				func4-en-mask = <0       0       0       0
+						 0       0       0       0      >;
+				volt-sel =      <NO_FUNC NO_FUNC NO_FUNC NO_FUNC
+						 NO_FUNC NO_FUNC NO_FUNC NO_FUNC >;
+				volt-sel-mask = <0       0       0       0
+						 0       0       0       0      >;
+				#pinmux-cells = <2>;
+				/delete-property/ gpio-group; /* KSI[7:0] */
```

Comment:
Note - you don't need to use `/delete-property/` here.  You just need to omit the `gpio-group`.